builtin/providers/aws/resource_aws_elasticache_replication_group.go

```diff
@@ -192,7 +195,17 @@ func resourceAwsElasticacheReplicationGroupRead(d *schema.ResourceData, meta int
 		return nil
 	}
 
-	d.Set("automatic_failover_enabled", rgp.AutomaticFailover)
+	if rgp.AutomaticFailover != nil {
+		switch strings.ToLower(*rgp.AutomaticFailover) {
+		case "disabled", "disabling":
+			d.Set("automatic_failover_enabled", false)
+		case "enabled", "enabling":
+			d.Set("automatic_failover_enabled", true)
+		default:
+			return fmt.Errorf("Unknown AutomaticFailover state %s", *rgp.AutomaticFailover)
```

Comment:
minor nit/question: do you think we should abort the import process in this case, or just log the warning?